src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProject_IWorkspaceProjectContext.cs

```diff
@@ -90,6 +90,20 @@ private void PostSetOptions(CommandLineArguments commandLineArguments)
                 var objOutputPath = PathUtilities.CombinePathsUnchecked(commandLineArguments.OutputDirectory, commandLineArguments.OutputFileName);
                 SetObjOutputPathAndRelatedData(objOutputPath);
             }
+
+            // TODO: Per https://github.com/dotnet/roslyn/issues/14584,
+            // We should consider switching to using commandLineArguments.ResolveMetadataReferences()
+            // For now, just use dynamic to add whatever 'mscorlib' or 'System.Runtime', the arguments
+            // would use.
+            if (Language == LanguageNames.VisualBasic)
+            {
+                dynamic dynArgs = commandLineArguments;
+                var coreLibReference = dynArgs.DefaultCoreLibraryReference as CommandLineReference?;
+                if (coreLibReference != null)
+                {
+                    AddMetadataReference(coreLibReference.Value.Reference, coreLibReference.Value.Properties);
```

Comment:
A concern I have with this is whether it's safe to call multiple times if the options change.  This isn't an issue in DPL, because they never change there, but may be an issue for CPS?